types/api/order.d.ts

```diff
@@ -0,0 +1,159 @@
+import { Part } from "../base";
+
+export type OrderState = {
+    id: number;
+    enabled: boolean;
+    description?: string;
+    created: number;
+    modified: number;
+    meta: Record<string, unknown>;
+    brand_t?: string;
+    retailer_t?: string;
+    vendor_t?: string;
+    name?: string;
+    justification?: string;
+    timestamp: number;
+    email: string;
+};
+
+export type Attachment = {
```

Comment:
At least name and kind are mandatory: https://github.com/ripe-tech/ripe-core/blob/master/src/ripe_core/models/attachment.py#L57

I know it's a pain but it's worse if we do this later when we're working on something unrelated, so please double-check the "optionality" for other models.

Reply:
Ah, well, I was checking on the `appier.field` declarations and since `mandatory=False` I thought they were optional. Forgot to check the validate. Thank you